src/App.scss

```
@@ -1 +1,36 @@
-// styles go here
+.App {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.Clock{
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 500px;
+  margin-bottom: 20px;
+  & p {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+  & span{
```

```suggestion
  margin-bottom: 20px;
  
  & p {
    display: flex;
    flex-direction: column;
    align-items: center;
  }
  
  & span{
```
Так трохи краще читається